constants.js

```diff
@@ -105,6 +105,9 @@ const constants = {
 
     // user metadata header to set object locationConstraint
     objectLocationConstraintHeader: 'x-amz-meta-scal-location-constraint',
+    /* eslint-disable camelcase */
```

Comment:
you can do /* eslint-disable-next-line camelcase */ and then you don't have to enable it again :)